smarty/templates/500.tpl

```diff
@@ -1,6 +1,4 @@
 <div class="container">
-{foreach from=$error_message item=message}
-    <h2>{$message}</h2>
-{/foreach}
```

Comment:
The issue is that it's inconsistent with https://github.com/aces/Loris/blob/15c0a2790843b089771e84fe8adf7ede2b53e750/src/Http/Error.php#L50 (which uses `message` instead of `error_message`).

Do you prefer to change the `Error` class to use `error_message` or should the other calling code across LORIS be changed?

Reply:
I don't actually know how many calling places there are, but wouldn't it be easier to change that to    `'message' => [$message]` instead of trying to hunt down the other ones?

Reply:
If you mean change the `Error.php` code, then I would need to change the `403` and `404` templates as well as their calling code instead.

Reply:
I don't have a preference but they need to be standardized one way or another if they are going to use `src/Http/Error.php`.